server/server.go

```diff
@@ -2712,10 +2718,15 @@ func readInputCompilePostV1(r io.ReadCloser) (*compileRequest, *types.ErrorV1) {
 		}
 	}
 
+	options := compileRequestOptions{
+		DisableInlining: request.Options.DisableInlining,
```

Comment:
🤔 It's not entirely clear to me why it's safe to access `request.Options.DisableInlining` -- do we know that `request.Options` can't be `nil`? Ah, that's because the field is `struct{}` not `*struct{}`, hm? OK.

We could inline it, then:

```go
	result := &compileRequest{
		Query:    query,
		Input:    input,
		Unknowns: unknowns,
		Options:  compileRequestOptions{
				DisableInlining: request.Options.DisableInlining,
		},
	}